app/embeds/GoogleMaps.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from "react";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+    "^https?://(?:www).google.com/maps/embed?pb=(.*)$"
```

Comment:
It seems like this is quite hard to get, you'd need to go to Share -> Embed in Google Maps and then copy and paste the html and grab the link out of it – is that correct?